docs/pages/business-and-security-contacts.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Business & Security Contacts
+description: Configure business and security contact emails to receive important updates from Teleport.
+tocDepth: 3
+---
+
+The **Business & Security Contacts** feature allows administrators to
+configure email addresses that will receive important operational,
+billing, and security-related notifications from Teleport.
+
+This helps ensure that the right teams in your organization stay
+informed about critical updates, such as license renewals,
+security advisories, and compliance-related changes.
+
+When you create a Teleport Enterprise account, the email associated with
+your administrator will automatically be set as a business and
+security contact.
+
+## Contact Types
+
+You can configure two types of contact emails:
+
+- **Business Contact**: Intended for billing, account management, and administrative notices.
+- **Security Contact**: Intended for vulnerability disclosures, security alerts, and compliance communication.
+
+You may configure the same email address for both roles or provide
+separate addresses based on your organization’s structure.
+Each contact type supports up to **three** email addresses.
+
+## Listing Contacts
+
+This section explains how to view the configured Business and Security contacts for a cluster.
+
+Currently, contacts can only be viewed and managed through the **Teleport Web UI**.  
+Other tools like `tctl` and the API are not supported for contact operations.
+
+Users will need permission to `list` the `contact` resource. On dashboards, all users have this permission.
+
+### Teleport Cloud
+
+To view or manage contacts for a cluster in Teleport Cloud, follow these steps:
+
+1. Go to https://teleport.sh and enter your tenant name.
+1. Sign in using your credentials.
+1. Hover over Zero Trust Access on the side menu and access Manage Clusters.
```

Comment:
Not an item for today, but we should change this some day. Business contacts shouldn't be buried this deep in the UI, and they probably shouldn't be under ZTA.